external/CMakeLists.txt

```diff
@@ -38,11 +38,18 @@
 #  - cmake ../..
 # Note: on MSVC, run startupMSVC.bat after deleting the build folder.
 
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.7)
```

Comment:
why the bump? Or keep this matching the other CMakes

Reply:
I noticed that this was not the same as the others so I bumped it.
Actually, MSVC 2019 requires CMake 3.14 (the latest) but nothing else requires that so I just commented in setupMSVC.bat that to use MSVC 2019 they had to install CMake 3.14.

One thing I found was that starting with MSVC 2019 the PLATFORM (i.e. the bitness) is set in a new runtime argument -A.  With MSVC 2017 the platform was part of the Generator name.  So the CMake command is now different between the two.

Also, the default bitness (if platform is not specified) was set to be 32bit in MSVC 2017 because it sorted before 64 bit.  https://aras-p.info/blog/2017/03/23/How-does-Visual-Studio-pick-default-config/platform/  With MSVC 2019 that seems to have changed.  I have both MSVC 2017 and 2019 installed.

So with some of the changes I made this time I hope it will correctly force 64bit in both compilers.

Reply:
> Actually, MSVC 2019 requires CMake 3.14 (the latest)

that is a reasonable compromise. Many distros do not ship that new cmake, eg latest Ubuntu LTS is at 3.10, so better when we're conservative.